core/widgetdiv.js

```diff
@@ -75,6 +77,7 @@ exports.getDiv = getDiv;
 /**
  * Allows unit tests to reset the div.
  * @param {?Element} newDiv The new value for the DIV field.
+ * @alias Blockly.WidgetDiv.testOnly_setDiv
```

Comment:
Given that this is test only we might want to omit it?

Reply:
I've added a `@ignore` annotation.